game/static/game/css/theme.css

```
@@ -268,7 +272,11 @@
 
 .nav-links {
     display: flex;
+    align-items: center;
     gap: 2rem;
+
+    flex: 1;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)